StylingAnalyzers.targets

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <ItemGroup>
+    <PackageReference Include="SonarAnalyzer.CSharp.Styling" Version="9.24.0.89429">
```

Comment:
We cannot use "*" due to json lock files. Each release of sonar-dotnet would break build of all other projects.

We will let update it manually, or let renovate do the job for us.